docs/appendices/release-notes/5.6.0.rst

```diff
@@ -82,7 +82,9 @@ None
 Performance and Resilience Improvements
 ---------------------------------------
 
-None
+- Tuned the behavior of the ``ANALYZE`` statement and the periodic stats update.
+  The operations should now require fewer reads from disk and the rate limiting
+  via the ``stats.service.max_bytes_per_sec`` setting should work better.
 
```

Comment:
I wonder if this also can be seen both as a fix and as an optimization (ie we can add 2 entries for 5.6 and for 5.1 mention only fix)?

From the user perspective it doesn't matter whether we have optimized version or not - ideally properly working throttling should take to account all details and since throttling didn't take to account readers reloading it was basically buggy?

Reply:
See also my reply here: https://github.com/crate/crate/issues/15072#issuecomment-1821014864
The rate limit is in general working.

I actually kinda struggle to confirm whether this PR improves the behavior at all.